File: OCH_Tao/ABC/A/206.js
const N = parseInt((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0], 10);
const X = Math.floor(N * 1.08);
if (X < 206) {
  console.log("Yay!");
} else if (X === 206) {
  console.log("so-so");
} else {
  console.log(":(");
}

File: OCH_Tao/ABC/A/206.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  int N, X;
  cin >> N;
  X = floor(N * 1.08);
  cout << (X < 206 ? "Yay!" : X == 206 ? "so-so" : ":(") << endl;
}